package.json

```diff
@@ -23,7 +23,8 @@
     "url": "http://github.com/pipedrive/client-nodejs.git"
   },
   "scripts": {
-    "test": "./node_modules/mocha/bin/mocha ./test/unit"
+	"test": "./node_modules/mocha/bin/mocha ./test/unit",
```

Comment:
@frozan @RuTsEST 
I remember that platform specifics used to be a problem, but this was a long time ago. It makes no sense to have a separate command for Windows and honestly I have not seen this used anywhere.

I think `node ./node_modules/mocha/bin/mocha ./test/unit` might work (https://shapeshed.com/writing-cross-platform-node/) on both. I don't have Windows to test.

Reply:
Yes, you are right about the path. I was using a Windows OS when I pulled this repo and ran into that path error but it worked fine when I ran it on Ubuntu. I thought to specify it for future reference.